R/input-daterange.R

```diff
@@ -118,6 +122,10 @@ dateRangeInput <- function(inputId, label, start = NULL, end = NULL,
         tags$input(
           class = "form-control",
           type = "text",
+          # `aria-labelledby` attribute is required for accessibility to avoid doubled labels (#2951).
+          `aria-labelledby` = paste0(inputId, "-label"),
+          # title attribute is announced for screen readers for date format.
+          title = paste("format:", format),
```

Comment:
Would it be helpful for this to be `Date format:` instead of `format:`? That is, is it already obvious that this is a _date_ input?